data.sql

```diff
@@ -72,4 +72,43 @@ SET owner_id = 4
 WHERE name in ('Charmander', 'Squirtle', 'Blossom');
 UPDATE animals
 SET owner_id = 5
-WHERE name in ('Angemon', 'Boarmon');
\ No newline at end of file
+WHERE name in ('Angemon', 'Boarmon');
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('William Tatcher', 45, '2000-04-23'),
+  ('Maisy Smith', 26, '2019-01-17'),
+  ('Stephanie Mendez', 64, '1981-05-04'),
```

Comment:
- [ ] I think you made a mistake in vet `Stephanie Mendez` date of graduation. It should be `1981-05-17`. Please try to update it.